systemd/ceph-radosgw@.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Ceph rados gateway
+After=network-online.target local-fs.target
+Wants=network-online.target local-fs.target
+PartOf=ceph.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/ceph
+Environment=CLUSTER=ceph
+ExecStart=/usr/bin/radosgw -f --conf /etc/ceph/${CLUSTER}.conf --name client.radosgw.%i
```

Comment:
I'm saying if you do "-C ${CLUSTER}" ceph will substitute it in in all the 
other places (like the default config file /etc/ceph/$cluster.conf, but 
also lots of other places).